lib/api/completeMultipartUpload.js

```diff
@@ -155,19 +164,59 @@ function completeMultipartUpload(authInfo, request, log, callback) {
             }
             return next(errors.MalformedXML, destBucket);
         },
-        function retrieveParts(destBucket, objMD, mpuBucket, jsonList, next) {
-            services.getMPUparts(mpuBucket.getName(), uploadId, log,
+        function getMPUMetadata(destBucket, objMD, mpuBucket, jsonList, next) {
+            return metadata.getObjectMD(mpuBucket.getName(), mpuOverviewKey,
+            {}, log, (err, storedMetadata) => {
+                if (err) {
+                    return next(err, destBucket);
+                }
+                const location = storedMetadata.controllingLocationConstraint;
+                return next(null, destBucket, objMD, mpuBucket, jsonList,
+                storedMetadata, location);
+            });
+        },
+        function ifMultipleBackend(destBucket, objMD, mpuBucket, jsonList,
+        storedMetadata, location, next) {
+            if (config.backends.data === 'multiple') {
+                return multipleBackendGateway.completeMPU(objectKey,
+                uploadId, location, jsonList, log, (err, newObjData) => {
```

Comment:
It's the object returned from AWS on a complete. It includes the ETag, bucket name, key name, location url, version id, and expiration if there's any lifecycle settings on the bucket.